second-edition/src/ch08-01-vectors.md

```diff
@@ -82,23 +82,23 @@ annotated in Listing 8-4:
 } // <- v goes out of scope and is freed here
 ```
 
-<span class="caption">Listing 8-4: Showing where the vector and its elements
-are dropped</span>
+<span class="caption">Listando 8-4: Mostrando donde el vector y sus elementos
+son dejados caer</span>
```

Comment:
'dejados caer' -> 'descartados' 

Please consider revising where else is this word used